### PR TITLE
Added purifycss to reduce CSS bundle sizes

### LINKED ISSUE #12: Add purifycss to the project
# Context

As we want to improve asset sizes on code files, we should integrate [PurifyCSS](https://github.com/purifycss/purifycss) in to the project.

## Acceptance Criteria

- Should be added using NPM
- Should have a watch task with gulp

### PR DESCRIPTION
**Pull request by TheMightyPenguin**
# Context

This was added to remove unused style rules, so the final CSS bundle will be smaller. A gulp task was implemented to purify the main css file. This was done by:

- Added gulp and gulp-purifycss dependencies
- Added gulp purify-css task, and watch task

This closes #12 

## Links

* [gulp](http://gulpjs.com/) for running tasks
* [gulp-purifycss](https://github.com/purifycss/gulp-purifycss)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,14 @@ This is the landing page for Simian CI.
 
 ## SASS
 
+_Make sure you run `jekyll serve` at least once before running any gulp task, as they depend on the that are on the _site folder_
```
Comment (duranmla) [inline README.md]: "on the that"

File: README.md [hunk 2]
```diff
@@ -12,8 +12,14 @@ This is the landing page for Simian CI.
...
+
 Jekyll compiles SASS when serving/building. Put your partials in the `_sass` folder, and your `main.scss` file in `css` folder. Your `main.scss` file should have the front matter declaration (dashes at the top of the file) to work correctly.
 
+We are using PurifyCSS to remove unused style rules, we made a gulp task for it, to start te watcher run:
```
Comment (duranmla) [inline README.md]: This is cool but I want some place that says like: "before start to develop run BLA and BLE to see the changes" and if we need to do something extra to see changes made on JS please clarify too

**Review by duranmla — CHANGES_REQUESTED** [review]
Few improvements to the docs @TheMightyPenguin

File: README.md (hunk 1 above)
Comment (TheMightyPenguin) [reply README.md]: oops! missed a word! ty

**Review by duranmla — APPROVED** [review]
LGTM

**Pull request merged**